src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -86,6 +88,29 @@ protected internal override void OpenPin(int pinNumber)
                 try
                 {
                     File.WriteAllText(Path.Combine(GpioBasePath, "export"), pinOffset.ToString(CultureInfo.InvariantCulture));
+
+                    // test to see if we have rw access to the GPIO pin.
+                    if (Interop.access(pinPath, AccessModeFlags.R_OK | AccessModeFlags.W_OK) != 0)
+                    {
+                        retryStopwatch = new Diagnostics.Stopwatch();
```

Comment:
`using System.Diagnostics` and use just type name

Reply:
Also you can use Stopwatch.StartNew();

Reply:
Changed in the latest update for the PR